src/shogun/machine/LinearMachine.cpp

```diff
@@ -120,3 +138,54 @@ void CLinearMachine::store_model_features()
 {
 }
 
+float64_t CLinearMachine::compute_bias()
+{
+	if (!m_labels)
+		SG_ERROR("No labels set\n")
+
+	if (!features)
+		SG_ERROR("No features set\n")
+
+	if (m_labels->get_num_labels() != features->get_num_vectors())
+		SG_ERROR("Number of training vectors does not match number of labels\n")
```

Comment:
Please always state the current values in such error messages. See
https://github.com/shogun-toolbox/shogun/wiki/Assertions